extensions/guacamole-auth-ldap/src/main/java/org/apache/guacamole/auth/ldap/ObjectQueryService.java

```diff
@@ -178,6 +169,9 @@ public String generateQuery(String filter,
      *
      * @param query
      *     The LDAP query to execute.
+     * 
+     * @param searchHop
+     *     The level of depth for this search, used for tracking referrals.
```

Comment:
Reading this description, I'm somewhat confused about what `searchHop` is. Is this the current depth of the search? Or the maximum depth? I think this should be made a bit clearer.

The code within the function looks like the former.

Reply:
Clarified this.